### ISSUE TITLE
Error on yq when using atlantis.yaml

### ISSUE THREAD

**Issue by jontreynes** [terraform-docs/gh-actions#31]
### Describe the bug
```
Error: unknown command "r" for "yq"
Run 'yq --help' for usage.
::debug file=entrypoint.sh,line=54 No files changed, skipping commit
```

### How can we reproduce it?

```
When using atlantis.yaml and
- name: Generate TF docs
  uses: terraform-docs/gh-actions@v0.6.0
  with:
    atlantis-file: atlantis.yaml
```

**Comment by khos2ow** [comment]
Thank you @jontreynes for reporting, I'll have a fix for this soon.

**Comment by khos2ow** [comment]
This is fixed in #34 and I'll be releasing `v0.7.0` soon, but in the meantime, if you're blocked by this, you can try the `main` (i.e. `uses: terraform-docs/gh-actions@main`)
